src/applications/vaos/new-appointment/components/VAFacilityPage/EligibilityCheckMessage.jsx

```diff
@@ -3,16 +3,26 @@ import { Link } from 'react-router-dom';
 import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox';
 import ErrorMessage from '../../../components/ErrorMessage';
 import FacilityAddress from '../../../components/FacilityAddress';
+import { ELIGIBILITY_REASONS } from '../../../utils/constants';
 
 export default function EligibilityCheckMessage({
   eligibility,
   facilityDetails,
+  typeOfCare,
   typeOfCareName,
 }) {
-  if (eligibility.requestFailed) {
+  const requestReason = eligibility.requestReasons[0];
+  const monthRequirement = facilityDetails?.legacyVAR?.settings
```

Comment:
We want to get the actual month limits from the VATS settings, since the VAOS service won't have those values in the endpoint we're using.

This is tricky to handle on the old facility page for partial Cerner users, so in that case I'm hedging and saying "12-24" months. That will affect very few users.